metrics/CryptographyEncryptionKeyManagement/SecurityReaderRoleAssigned/metric.rego

```diff
@@ -0,0 +1,72 @@
+package cch.metrics.security_reader_role_assigned
+
+import data.cch.compare
+import rego.v1
+import input as assignments
+
+default applicable = false
+
+default compliant = false
+
+applicable if {
+    security_reader_assigned_users_available
+}
+
+applicable if {
+    flag := security_reader_role_flag
+    flag == true
+}
+
+applicable if {
+    flag := security_reader_role_flag
+    flag == false
```

Comment:
Why is it applicable if the role flag is true AND it is applicable if it is false?